ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricalStreamObserver.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool.metrics
+
+import akka.actor.typed.ActorRef
+import org.slf4j.Logger
+
+class MetricalStreamObserver[T](
```

Comment:
The word metrical sounds a bit too poetic. Elsewhere we use the word Metered or Timed in similar contexts.
```suggestion
class MeteredStreamObserver[T](
```

Reply:
I would say that the name `MeteredStreamObserver` suggests that the observer itself is metered but I'm not 100% sure :)
If you're convinced to the `MeteredStreamObserver` I will change the name.